### ISSUE TITLE
RawShaderMaterial in Three.js 0.154.1 appears to have a regression

### ISSUE THREAD

**Issue by DougReeder** [aframevr/aframe#5339]
**Description:**
When instantiating a RawShaderMaterial, this error is logged:

three.module.js:19914 THREE.WebGLProgram: Shader Error 0 - VALIDATE_STATUS false

Program Info Log: Vertex shader is not compiled.
�
VERTEX

ERROR: 0:3: 'version' : #version directive must occur before anything else, except for comments and white space
ERROR: 0:4: 'in' : storage qualifier supported in GLSL ES 3.00 and above only
ERROR: 0:5: 'in' : storage qualifier supported in GLSL ES 3.00 and above only
ERROR: 0:8: 'out' : storage qualifier supported in GLSL ES 3.00 and above only
�
```
  1: #define SHADER_TYPE RawShaderMaterial
  2: #define SHADER_NAME 
> 3: #version 300 es
  4: in vec2 uv;
  5: in vec3 position;
  6: uniform mat4 projectionMatrix;
  7: uniform mat4 modelViewMatrix;
  8: out vec2 vUV;
  9: void main(void) {
```
FRAGMENT

ERROR: 0:3: 'version' : #version directive must occur before anything else, except for comments and white space

- A-Frame Version: master 1.4.2 (Date 2023-07-13, Commit #aacacadd)
- Platform / Device: Chrome on MacOS
- Reproducible Code Snippet or URL: showcase/model-viewer/ in master

**Comment by dmarcos** [comment]
Fixed by https://github.com/aframevr/aframe/pull/5328

**Comment by sigurof** [comment]
Exactly which version is this fixed in? I am getting the same error in `0.160.0`.

```
rogram Info Log: Vertex shader is not compiled.

VERTEX

ERROR: 0:3: 'version' : #version directive must occur before anything else, except for comments and white space
ERROR: 0:10: 'out' : storage qualifier supported in GLSL ES 3.00 and above only
ERROR: 0:11: 'out' : storage qualifier supported in GLSL ES 3.00 and above only
ERROR: 0:16: 'switch' : Illegal use of reserved word
ERROR: 0:16: 'switch' : syntax error


  1: #define SHADER_TYPE RawShaderMaterial
  2: #define SHADER_NAME 
> 3: # version 300 es
  4: 
  5: uniform float aspect;
  6: uniform float radius;
  7: uniform vec2 center;
  8: uniform vec3 color;
  9: 
```

**Comment by sigurof** [comment]
For anyone stumbling over this looking for an answer to why your ThreeJs RawShaderMaterial refuses to compile. 
You need to set the ` glslVersion: THREE.GLSL3` as one of the configuration properties when instantiating.

```
    const material = new THREE.RawShaderMaterial({
        glslVersion: THREE.GLSL3,
        vertexShader: myVtxShader,
        fragmentShader: myFragShader,
```
